src/components/TimeCardPage/TimeSheet.tsx

```diff
@@ -1,279 +1,320 @@
+    FirstName: "crys",
+    LastName: "tal",
+    Type: "Employee",
+    Picture: "https://www.google.com/capybara.png",
+  },
+  {
+    UserID: "def",
+    FirstName: "ken",
+    LastName: "ney",
+    Type: "Employee",
+    Picture: "https://www.google.com/koala.png",
+  },
+];
+
+function SearchEmployeeTimesheet({ employees, setSelected }) {
+  const handleChange = (selectedOption) => {
+    setSelected(selectedOption);
+  };
+
+  const customStyles = {
+    control: (base) => ({
+      ...base,
+      flexDirection: "row-reverse",
+    }),
+  };
+
+  const DropdownIndicator = (props) => {
     return (
-        <Card direction="row" width="50%">
-            <Avatar src={employee?.Picture} name={employee?.FirstName + " " + employee?.LastName} size='md' showBorder={true} borderColor='black' borderWidth='thick'/>
-            <CardBody>
-                <Text>{employee?.FirstName + " " + employee?.LastName}</Text>
-            </CardBody>
-        </Card>
-    )
-}
-
-function SearchEmployeeTimesheet({employees, setSelected}) {
-    
-    const handleChange = (selectedOption) => {
-        setSelected(selectedOption);
-    }
-
-    const customStyles = {
-        control: (base) => ({
-          ...base,
-          flexDirection: 'row-reverse',
-        }),
-      }
-
-    const DropdownIndicator = (props) => {
-        return (
-          <components.DropdownIndicator {...props}>
-            <SearchIcon />
-          </components.DropdownIndicator>
-        );
-      };
-
-    // TODO: fix styling
-    // at the moment defaultValue is the first user in the employees array
-    // which is currently an invariant that matches the useState in Page
-    return (
-        <div style={{width: '600px'}}>
-            <Select isSearchable={true} 
-            defaultValue={employees[0]} 
-            chakraStyles={customStyles} 
-            size="lg" 
-            options={employees} 
-            onChange={handleChange} 
-            components={{ DropdownIndicator }}
-            getOptionLabel={option =>`${option.FirstName + " " + option.LastName}`}
-            getOptionValue={option => `${option.FirstName + " " + option.LastName}`}/>
-        </div>
-    )
+      <components.DropdownIndicator {...props}>
+        <SearchIcon />
+      </components.DropdownIndicator>
+    );
+  };
+
+  // TODO: fix styling
+  // at the moment defaultValue is the first user in the employees array
+  // which is currently an invariant that matches the useState in Page
+  return (
+    <div style={{ width: "600px" }}>
+      <Select
+        isSearchable={true}
+        defaultValue={employees[0]}
+        chakraStyles={customStyles}
+        size="lg"
+        options={employees}
+        onChange={handleChange}
+        components={{ DropdownIndicator }}
+        getOptionLabel={(option) =>
+          `${option.FirstName + " " + option.LastName}`
+        }
+        getOptionValue={(option) =>
+          `${option.FirstName + " " + option.LastName}`
+        }
```

Comment:
🏕️🤔📌   I think I've seen this somewhere else too - could be a common utility function. As an extreme, you could even map it when you fetch the data so that there's a `FullName` property that you can just reference. Then you can take advantage of its accessibility without having to maintain the data integrity on edit. (you don't have to do this now, but I'm interested in your thoughts)

Reply:
I think its a good idea to make it into a common utility function and toss this function and other utility functions into a utils file under src